src/web3/networkConfig.ts

```diff
@@ -154,6 +155,10 @@ const AAVE_ARB_USDC_TOKEN_ID_MAINNET = `${NetworkId['arbitrum-one']}:0x724dc807b
 const ARB_AAVE_POOL_V3_CONTRACT_ADDRESS_STAGING = '0xBfC91D59fdAA134A4ED45f7B584cAf96D7792Eff'
 const ARB_AAVE_POOL_V3_CONTRACT_ADDRESS_MAINNET = '0x794a61358D6845594F94dc1DB02A252b5b4814aD'
 
+// also called rewards controller
+const ARB_AAVE_INCENTIVES_V3_CONTRACT_ADDRESS_STAGING = '0x3A203B14CF8749a1e3b7314c6c49004B77Ee667A'
```

Comment:
These should be lowercase for consistency.

Reply:
I believe contract addresses in the file are currently all checksummed addresses